packages/components/src/make-localized-element.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Localized } from 'fluent-react/compat';
+
+export function formatVars(l10nVars = {}) {
+  const varNames = Object.keys(l10nVars);
+
+  return varNames.reduce(
+    (vars, name) => Object.assign({}, vars, { [`$${name}`]: l10nVars[name] }),
```

Comment:
- [ ] can you make sure that the babelrc setup compiles this down to include the polyfill (but not in the global manner that `@babel/polyfill` does?

https://babeljs.io/docs/en/babel-plugin-transform-runtime I think is the transform to use.

Reply:
Why not just use spreads here?

```js
(vars, name) => {...vars, [`$${name}`]: l10nVars[name] },
```

Reply:
https://github.com/udacity/fluent-react-utils/issues/20